src/data/ad.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "OpenWhisk and resin.io for edge computing",
-  "text": "A pair of writeups for using resin.io with OpenWhisk as a distributed compute service",
-  "link": "https://forums.resin.io/t/openwhisk-writeup/593",
-  "image": "https://cdn-images-1.medium.com/max/1200/1*EEJjDyoHUnuY90zm2d3Uqw.png"
+  "title": "Changes in resin.io support",
+  "text": "Our goal is for the entire resin.io community to become more knowledgeable with each issue solved and question answered.",
```

Comment:
let's preview all of the first paragraph from the forum post, so it's clear what the change actually is: _"We’re making a few changes to our support process. Our goal is for the entire resin.io community to become more knowledgeable with each issue solved and question answered. We’re therefore shifting our support focus to the place with the most impact and visibility: our forums."_

Also, can/should we also add a second link to resin.io/support?